app/controllers/application_controller.rb

```diff
@@ -99,6 +98,9 @@ def skip_sentry_exception_types
       else
         Common::Exceptions::InternalServerError.new(exception)
       end
+    
+    va_exception_info = { va_exception_errors: va_exception.errors.map(&:to_hash) }
+    log_exception_to_sentry(exception, extra.merge(va_exception_info)) unless skip_sentry_exception_types.include?(exception.class)
```

Comment:
Can we switch this to a block conditional rather than the postfix condition and include the data transform on line 102 in the block?

```
unless  skip_sentry_exception_types.include?(exception.class)
  va_exception_info = { va_exception_errors: va_exception.errors.map(&:to_hash) }
  log_exception_to_sentry(exception, extra.merge(va_exception_info))
end
```

That allows me to ignore both lines if I know the condition, rather than having to read the first line only to realize that `va_exception_info` will go unused.